torch/_functorch/aot_autograd.py

```diff
@@ -1800,22 +1802,15 @@ def add_dupe_args(args):
     deduped_flat_args = remove_dupe_args(flat_args)
 
     tracing_context = TracingContext.get()
-    if tracing_context:
+    if tracing_context and aot_config.aot_autograd_arg_pos_to_source and len(aot_config.aot_autograd_arg_pos_to_source) > 0:
```

Comment:
nit: an empty list will interpret as bool false. So you do not need the length test.

Reply:
This can be None